Tools/autotest/quadplane.py

```diff
@@ -103,6 +103,40 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_motor_mask(self):
+        """Check operation of output_motor_mask"""
+        if not(int(self.get_parameter('Q_TILT_MASK')) & 1):
+#        or (int(self.get_parameter('Q_TAILSIT_MOTMX')) & 1)
+            self.progress("output_motor_mask not in use")
+            return
+        self.progress("Testing output_motor_mask")
+        self.reboot_sitl()
+        self.wait_ready_to_arm()
+
+        self.progress('Assert that SERVO5 is Motor1')
+        assert(33 == self.get_parameter('SERVO5_FUNCTION'))
+
+        self.progress("Testing MANUAL mode")
+        self.mavproxy.send('mode MANUAL\n')
+        self.wait_mode('MANUAL')
+        self.arm_vehicle()
+        self.progress("Raising throttle to 1800")
+        self.set_rc(3, 1800)
+        self.progress("Waiting for Motor1 to start")
+        self.wait_servo_channel_exceed_value(5, 1100)
+
+        self.reboot_sitl()
+        self.wait_ready_to_arm()
+
+        self.progress("Testing FBWA mode")
+        self.mavproxy.send('mode FBWA\n')
+        self.wait_mode('FBWA')
+        self.arm_vehicle()
+        self.progress("Raising throttle to 1800")
+        self.set_rc(3, 1800)
+        self.progress("Waiting for Motor1 to start")
+        self.wait_servo_channel_exceed_value(5, 1100)
```

Comment:
Leaving the vehicle armed with motor spinning at the end of the test isn't good :-)

Please `self.zero_throttle()` and `self.disarm_motors()`

Reply:
A test should leave the vehicle in some specific state?

Reply:
Yes, disarmed on the ground.

If it's not then we will reboot the vehicle, which slows the test suite own.